addons/l10n_es_pos/tests/test_frontend.py

```diff
@@ -37,3 +37,12 @@ def get_number_of_regular_invoices():
         num_of_regular_invoices = get_number_of_regular_invoices() - initial_number_of_regular_invoices
         self.assertEqual(num_of_simp_invoices, 3)
         self.assertEqual(num_of_regular_invoices, 1)
+
+    def test_pos_session(self):
+        simplified_partner = self.env.ref("l10n_es.partner_simplified", raise_if_not_found=False)
+        if simplified_partner:
+            simplified_partner.unlink()
+        self.assertFalse(self.env.ref("l10n_es.partner_simplified", raise_if_not_found=False))
+        self.main_pos_config.with_user(self.pos_user).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'open_pos_session', login="pos_user")
```

Comment:
Instead of starting a tour for nothing you can just access the `simplified_partner_id` that will trigger the compute.
Like this `self.assertTrue(self.main_pos_config.simplified_partner_id)`